presto-product-tests/src/main/resources/tempto-configuration.yaml

```diff
@@ -21,6 +21,7 @@ databases:
     warehouse_directory_path: /user/hive/warehouse
     inject_stats_for_immutable_tables: true
     inject_stats_for_mutable_tables: true
+    create_tables_as_acid: false
```

Comment:
This property is weridly named. Based on usege it means "Hive is creating tables as ACID by default". Hence the name suggest "Tempto should create tables as ACID".
We should either flip the meaning.
Or name it sth like "orc_tables_created_as_acid_in_hive".

But I think we should just reveverse its meaning to "Tempto should create tables as non-transactional". 
And then use:
```
(createTablesAsAcid.orElse(false) ? "TBLPROPERTIES ('transactional' = 'true')" : "");
```

makes sense?